docs: restructure docs sidebar (#1796)
urbica/martin: docs/src/sources-resources.md

# Supporting Resources

> Data alone is not enough to create a map.

Reviewer: ```suggestion
Data alone is not enough to show an interactive map.
```
